plugins/woocommerce-docs/scripts/manifest.json

```diff
@@ -28,12 +28,12 @@
       "posts": [
         {
           "title": "Unit Testing",
-          "url": "https://raw.githubusercontent.com/woocommerce/woocommerce/add/wc-docs-manifest/plugins/woocommerce-docs/example-docs/testing/unit-tests.md",
+          "url": "https://raw.githubusercontent.com/woocommerce/woocommerce/dev/use-commonmark/plugins/woocommerce-docs/example-docs/testing/unit-tests.md",
           "id": "c9916c10ae1962ecaa4193ccdadd043241b8ec05"
         }
       ],
       "categories": []
     }
   ],
-  "hash": "7c6cdf9772ccf1f9327a35c0ef8e45413085225e20b1df503e2a19a444c106fb"
+  "hash": "bc62180b4d58b63e0f2053b6c16783fce64e339714b282df2c4c8fccf57cc689"
 }
```

Comment:
It would be good to reference this manifest on Github in the README as opposed to the local copy. Or at least make note of the `special Mac OS external docker host address` thing because I ran into the same issue described below

https://github.com/woocommerce/woocommerce/pull/38874#issuecomment-1607401368

Reply:
Hmm yeah probably this approach of hosting a file inside a plugin running in Docker is still useful for future development of the plugin. The only thing is that this will change again with my PR in progress here: https://github.com/woocommerce/woocommerce/pull/39120

How about if in that PR I update the README for the plugin to include instructions for running the manifest generator, hosting the file inside the plugin and also caveats for users of Docker with this approach (ie including the special docker host name)?

Reply:
I've got updated instructions in the other PR for generating manifest, I also added instructions for docker host issues you mentioned: https://github.com/woocommerce/woocommerce/pull/39120/files#diff-e6483b5a793df3870167230500ed1be3fadf62c3cf8d59d411fba3d134bc5bbe